Odata-docs/client/v7/batch-operations.md

```diff
@@ -85,42 +86,38 @@ The payload of the request is as following:
 ```
 ## Batch Modification 
 
-In order to batch a set of changes to the server, `ODataServiceContext` provides `SaveChangesOptions.BatchWithSingleChangeset` and `SaveChangesOptions.BatchWithIndependentOperations` when `SaveChanges`.
+In order to batch a set of changes to the server, `DataServiceContext` provides `SaveChangesOptions.BatchWithSingleChangeset` and `SaveChangesOptions.BatchWithIndependentOperations` when `SaveChanges`.
 
 `SaveChangesOptions.BatchWithSingleChangeset` will save changes in a single change set in a batch request.
```

Comment:
Worth explaining what exactly this means - that if one request in the batch fails they all fail?

Reply:
#Resolved